public/app/features/alerting/unified/components/silences/MatchedSilencedRules.tsx

```diff
@@ -48,7 +48,12 @@ export const MatchedSilencedRules = () => {
 
   return (
     <div>
-      <h4>Affected alerts {matchedAlertRules.length > 0 ? `(${matchedAlertRules.length})` : ''}</h4>
+      <h4 className={styles.title}>
+        Affected alerts
+        {matchedAlertRules.length > 0 ? (
+          <Badge className={styles.badge} color="blue" text={matchedAlertRules.length} />
+        ) : null}{' '}
```

Comment:
Prettier seems to love to add these `{' '}` 👀

Reply:
😩